docs/data/base/getting-started/customization/customization.md

```diff
@@ -0,0 +1,83 @@
+# Customizing MUI Base components
+
+<p class="description">There are several ways to customize MUI Base components, from applying custom CSS rules to building fully custom components using hooks.</p>
+
+With MUI Base, you have the freedom to decide how much you want to customize a component's structure and style.
+
+This document reviews the three levels of customization that are available: applying custom CSS rules, overriding default subcomponent slots, and using hooks to build fully custom components.
+
+## Applying custom CSS rules
+
+If you're happy with the default structure of a component's rendered HTML, you can apply custom styles to the component's classes.
+
+Each component has its own set of classes.
+Some classes are **static**, which is to say that they are always present on the component.
+Others are applied **conditionally**—like `Mui-disabled`, for example, which is only present when a component is disabled.
+
+Each component's API documentation lists all classes that the component uses.
+Additionally, you can import a `[componentName]Classes` object that describes all the classes a given component uses, as the following demo illustrates:
+
+{{"demo": "StylingCustomCss.js"}}
```

Comment:
The short preview doesn't show the most important bits. Let's show the expanded version.

```suggestion
{{"demo": "StylingCustomCss.js", "defaultCodeOpen": true}}
```